### PR TITLE
chore: unsupported flag error

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [X] Ready for review
- [X] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Add custom error for unsupported feature flags cases
Add test for that

### CODE REVIEW

File: src/lib/errors/unsupported-feature-flag-error.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {CustomError} from './custom-error';
+
+export class UnsupportedFeatureFlagError extends CustomError {
+    private static ERROR_CODE: number = 403;
+
+    constructor(featureFlag: string, userMessage = `Feature flag '${featureFlag}' is not enabled for your org`) {
```
Comment (orsagie) [inline src/lib/errors/unsupported-feature-flag-error.ts]: Maybe different wording like:
`Feature flag '${featureFlag}' is not currently enabled for your org, to enable please contact snyk support`
```suggestion
    constructor(featureFlag: string, userMessage = `Feature flag '${featureFlag}' is not currently enabled for your org, to enable please contact snyk support`) {
```

**Comment by adrukh** [comment]
@dkontorovskyy please note that the current commit messages will not create a new release, see https://github.com/snyk/snyk/pull/675#issuecomment-514104052

**Comment by dkontorovskyy** [comment]
@adrukh it's okey. next one will create a release

**Comment by dkontorovskyy** [comment]
@lili2311 but it's not adding any functionality actually. It's basically refactoring

**Pull request merged**